torch_glow/tests/nodes/sigmoid_test.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import torch
+import torch_glow
+
+from tests.utils import jitVsGlow
+
+# Basic test of the PyTorch sigmoid Node on Glow
+
+
+def test_sigmoid_basic():
+    def sigmoid_basic(a):
```

Comment:
I started just calling these `test_f` so I don't need to come up with a new name each time, you can do the same or not up to you